src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -102,29 +108,27 @@ def __push_records(
         self: "FeedbackDataset",
         client: "httpx.Client",
         id: UUID,
-        question_mapping: Dict[str, UUID],
+        question_name_to_id: Dict[str, UUID],
         show_progress: bool = True,
     ) -> None:
-        for i in trange(
-            0, len(self.records), PUSHING_BATCH_SIZE, desc="Pushing records to Argilla...", disable=show_progress
-        ):
-            try:
-                records = []
-                for record in self.records[i : i + PUSHING_BATCH_SIZE]:
-                    if record.suggestions:
-                        for suggestion in record.suggestions:
-                            suggestion.question_id = question_mapping[suggestion.question_name]
-                    records.append(
-                        record.dict(
-                            exclude={"id": ..., "suggestions": {"__all__": {"question_name"}}}, exclude_none=True
-                        )
+        if len(self.records) > 0:
```

Comment:
I would invert this condition to `if len(self.records) == 0: return`, so we don't indent the code that much